src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
Please don't "using namespace bla".  If it's in a namespace the point is to keep it distinct, just prepend the namespace before invocations of elements from that namespace.